webkit2/jscore.jsc-exception.lisp

```diff
@@ -64,3 +64,13 @@
 (defcfun "jsc_exception_report" :string
   (exception (g-object jsc-exception)))
 (export 'jsc-exception-report)
+
+(define-condition jsc-exception-condition (error)
+  ((exception :initform (error "There should be a JSCException.")
+              :accessor exception
+              :initarg :exception))
+  (:documentation "Condition to signal when a JSCException is raised.")
+  (:report (lambda (condition stream)
+             (format stream "~a~&~%~%~a"
```

Comment:
Why not just "~&~&"?

Reply:
Hmmmm...

Reply:
Simplified in e213c8ff3fbe962cf0d893644e3ce71509760547.